crates/solver/src/metrics.rs

```diff
@@ -184,6 +187,13 @@ impl Metrics {
         })?;
         registry.register(Box::new(transaction_submission.clone()))?;
 
+        let opts = Opts::new(
+            "transaction_gas_price",
```

Comment:
BTW there's a [naming convention for prometheus metrics][1], and it requires names to include units. So I guess this should be `transaction_gas_price_gwei`.

[1]: https://prometheus.io/docs/practices/naming/